### PR TITLE
Changed Typo Mistakes

### PR DESCRIPTION
**Pull request by Dheeraj-2001**
Changes made to

```diff
-Behavior
+Behaviour
```

### CODE REVIEW

**Review by RonenTGreat — COMMENTED** [review]
If you are speaking the British English(behaviour) then it would be a typo but that's the spelling in American English (behavior).

**Review by CodingSpecies — COMMENTED** [review]
I think this repo is based on British spelling though. @Dheeraj-2001 @RonenTGreat

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -50,15 +50,15 @@ decisions when appropriate.
 
 ## Scope
 
-This Code of Conduct applies within all community spaces, and also applies when
+This Code of Conduct applies within all community spaces and also applies when
 an individual is officially representing the community in public spaces.
-Examples of representing our community include using an official e-mail address,
+Examples of representing our community include using an official e-mail address,
```
Comment (schmelto) [inline CODE_OF_CONDUCT.md:55]:
```suggestion
Examples of representing our community include using an official e-mail address,
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -72,19 +72,19 @@ the consequences for any action they deem in violation of this Code of Conduct:
 
 ### 1. Correction
 
-**Community Impact**: Use of inappropriate language or other behavior deemed
```
Comment (krshdev) [inline CODE_OF_CONDUCT.md:75]: Actually this is correct spelling on behavior Ig?

**Review by krshdev — CHANGES_REQUESTED** [review]
I think we should keep the default COC!

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (nhcarrigan) [reply CODE_OF_CONDUCT.md:75]: "behavior" is American English, "behaviour" is British English

**Pull request closed without merging**